packages/FormElement/stories/examples/Fieldset.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Rule } from "storybook/assets/styles/common.styles";
+import L10n from "@paprika/l10n";
+import Input from "@paprika/input";
+import Select from "@paprika/select";
+import Button from "@paprika/button";
+import FormElement, { Fieldset } from "../../src";
+
+const { Label, Instructions, Description, Content, Error } = FormElement;
+
+export default function EverythingExample() {
+  const [value, setValue] = React.useState();
+  const [value2, setValue2] = React.useState();
+  const [errorText, setErrorText] = React.useState();
+
+  return (
+    <L10n>
+      <Fieldset>
+        <Label>Fieldset label</Label>
+        <Instructions>These are some fieldset instructions.</Instructions>
+        <Content>
+          <div
+            css={`
+              display: flex;
+              width: 100%;
+            `}
+          >
+            <div
+              css={`
+                width: 20%;
+              `}
+            >
+              <FormElement>
+                <Label isVisuallyHidden>Salutation</Label>
```

Comment:
Wonder if the `isVisuallyHidden` functionality should be handled by the component if it is wrapped in a `FieldSet` rather than leaving the consumer to add it. How do they know if it should or shouldnt' be only showing for screen readers? Perhaps we should encourage them to always add `Labels` by default. But the component can decided whether it should be showing or not to the user.

Reply:
Oh I think it has to depend on the design.  Sometimes the labels should show, sometimes only _one_ should show, sometimes they can be hidden if the `<Fieldset.Label>` is sufficient.  I think this is still the best way to provide necessary flexibility.